generate-wasm-examples/generate_wasm_examples.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/sh
 
+# Print all executed lines
+# set -x
+
 git init bevy
 cd bevy
 git remote add origin https://github.com/bevyengine/bevy
+git restore Cargo.toml
```

Comment:
why is this needed?

Reply:
Because of the `echo >> Cargo.toml` :D I'll be glad to add it to the main repo `Cargo.toml`